scaling_performance/optimizing_storage.adoc

```diff
@@ -322,7 +343,7 @@ loopback mounted sparse file. To see if your system is using the loopback mode:
 
 [IMPORTANT]
```

Comment:
Would this make more sense at 240?

If you know why you want to use overlay2, I'd say that instead of "Red Hat strongly recommends."